blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -46,6 +46,8 @@ Breaking Changes
 Changes
 =======
 
+- Added support for executing aggregations as ``window functions``, currently
+  supporting the empty ``OVER`` clause to define the ``window``.
```

Comment:
I'd avoid the `currently`, e.g.: Added support execution aggregations as ``window functions`` with an empty ``OVER`` clause as the ``window`` definition.